pkg/database/signing_key.go

```diff
@@ -40,6 +42,34 @@ func (s *SigningKey) GetKID() string {
 	return fmt.Sprintf("r%dv%d", s.RealmID, s.ID)
 }
 
+func (s *SigningKey) ManagedKeyID() string {
+	return s.KeyID
+}
+
+func (s *SigningKey) IsActive() bool {
+	return s.Active
+}
+
+func (s *SigningKey) SetRealmID(id uint) {
+	s.RealmID = id
+}
+
+func (s *SigningKey) SetManagedKeyID(keyID string) {
+	s.KeyID = keyID
+}
+
+func (s *SigningKey) SetActive(active bool) {
+	s.Active = active
+}
+
+func (s *SigningKey) Table() string {
```

Comment:
Does passing in as `Model` not work? It might not, I've never tried.

Reply:
will mess w/ that separate